frontend-v2/src/features/results/Results.tsx

```diff
@@ -1,8 +1,9 @@
-import { Tabs, Tab, Box } from "@mui/material";
-import Add from "@mui/icons-material/Add";
-import { ChangeEvent, FC, useState } from "react";
-
+import { Tabs, Tab, Box, Button, IconButton } from "@mui/material";
+import RemoveCircleOutlineIcon from "@mui/icons-material/RemoveCircleOutline";
+import { ChangeEvent, FC, useEffect, useState } from "react";
+import ArrowDropDownIcon from "@mui/icons-material/ArrowDropDown";
 import ResultsTab from "./ResultsTab";
+import { TableHeader } from "../../components/TableHeader";
```

Comment:
Not a blocker for this PR, but it's just occurred to me that this name is slightly confusing, since there's also a [MUI `TableHead` component](https://mui.com/material-ui/api/table-head/) and the `TableHeader` component has nothing to do with data tables.

Reply:
I will be creating another mr soon, possibly even today, with Export functionality. I will rename this component there, to not block this page from being tested